tmt/export.py

```diff
@@ -456,6 +540,13 @@ def export_to_nitrate(test):
         echo(style(
             'Add migration warning to the test case notes.', fg='green'))
 
+    # UUID
+    uuid = add_uuid_if_not_defined(test.node, dry=dry_mode)
+    if not uuid:
+        uuid = test.node.get(ID_KEY)
+    struct_field.set('uuid', uuid)
```

Comment:
```suggestion
    struct_field.set('id', uuid)
```

Let's use naming consistent with the `id` attribute in `tmt` specification.